src/CribblyBackend.DataAccess/Exceptions/DivisionNotFoundException.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+
+namespace CribblyBackend.DataAccess
+{
+    public class DivisionNotFoundException : Exception
+    {
+        public DivisionNotFoundException()
+        {
+            
+        }
+
+        public DivisionNotFoundException(string message)
+            : base("Division not found")
+        {
+            
+        }
+
+        public DivisionNotFoundException(string message, Exception inner)
+            : base("Division not found", inner)
+        {
+            
+        }
```

Comment:
I think we want one constructor here that takes an ID and constructs a message with it

Reply:
fixed